authors/apps/authentication/views.py

```diff
@@ -75,6 +81,9 @@ def post(self, request):
 
 
 class UserRetrieveUpdateAPIView(RetrieveUpdateAPIView):
+    """
```

Comment:
can you add the docstrings to each method in the class as well, refer to the implementation used in the profiles app as a guide.  @Amoswachira

Reply:
@oluwagbenga-joloko okay let me update that

Reply:
@oluwagbenga-joloko I have added the docstrings